system/modules/playos-status.nix

```diff
@@ -0,0 +1,55 @@
+{ config, lib, pkgs, ... }:
+let
+  ttyNumber = 8;
+  tty = "tty${toString ttyNumber}";
+  ttyPath = "/dev/${tty}";
+  driverUrl = "http://127.0.0.1:8382";
+
+  script =
+    pkgs.writeShellScriptBin "print-status" ''
+      while :; do
+        screen=$(xrandr --current | grep '*' | awk '{print $1}')
+        networkCount=$(connmanctl services | grep wifi | wc -l)
+        rfid=$(opensc-tool --list-readers)
```

Comment:
Because this output has multiple lines the indentation below will only work on the first line. A quick fix could be piping into `pr -T -o 2` (from `coreutils`).

Reply:
Good idea, thanks!